src/crafting.cpp

```diff
@@ -601,57 +655,28 @@ void player::complete_craft()
         }
     }
 
+    std::list<item> used( craft.components.begin(), craft.components.end() );
```

Comment:
Looks like you transform from `vector` to `list` and vice-versa alot. Why not stick to one of them (preferably to `vector`) ?

Reply:
Only reason is that I was a little lazy about cleaning up the old code, will fix.